wiki/BanchoBot/de.md

```diff
@@ -6,19 +6,19 @@ outdated: true
 
 ![BanchoBots Benutzerpanel.](img/BanchoBot.jpg "BanchoBots Benutzerpanel.")
 
-BanchoBot ist ein Bot (d.h. er führt eine Liste an Befehlen aus und antwortet mit einem Ergebnis), der von **Echo49** entwickelt wurde. Er hilft Spielern im Chat, indem er nützliche Informationen anzeigt und wichtige spielbezogene Nachrichten verkündet. Er ist der [Host](https://de.wikipedia.org/wiki/Hostrechner) des [bancho irc](/wiki/Internet_Relay_Chat) und hat ein Nutzerprofil mit der ID [3](https://osu.ppy.sh/users/3) (neben peppy mit der ID [2](https://osu.ppy.sh/users/2)). _Außerdem hat er ein eigenes [Twitterprofil](https://twitter.com/banchoboat)!_
+BanchoBot ist ein Bot (d.h. er führt eine Liste an Befehlen aus und antwortet mit einem Ergebnis), der von **Echo49** entwickelt wurde. Er hilft Spielern im Chat, indem er nützliche Informationen anzeigt und wichtige spielbezogene Nachrichten verkündet. Er ist der [Host](https://de.wikipedia.org/wiki/Hostrechner) des [bancho irc](/wiki/Internet_Relay_Chat) und hat ein Nutzerprofil mit der ID [3](https://osu.ppy.sh/users/3) (neben peppy mit der ID [2](https://osu.ppy.sh/users/2)). *Außerdem hat er ein eigenes [Twitterprofil](https://twitter.com/banchoboat)!*
 
 ## BanchoBots Befehle
 
 Um Befehle von BanchoBot ausführen zu lassen, muss immer ein `!` am Anfang der Nachricht stehen. **Normale Nutzer können Befehle nur in Mehrspielerkanälen und in Privatnachrichten an BanchoBot ausführen!** Wenn du einen Befehl in öffentlichen Kanälen sendest, ist die Antwort darauf immer eine private Nachricht, die von anderen Nutzern nicht gesehen werden kann. Man kann auch den Clientbefehl `/bb <Befehl>` verwenden, um den Befehl direkt als Privatnachricht an BanchoBot zu senden.
 
-_Für eine Liste an Clientbefehlen (`/befehl`) siehe [Chatkonsole](/wiki/Chat_Console#commands-list)._
+*Für eine Liste an Clientbefehlen (`/befehl`) siehe [Chatkonsole](/wiki/Chat_Console).*
```

Comment:
why remove anchor (same with other languages) ?

update to the correct one if the translation exists, otherwise use the english one